src/hepattn/models/task.py

```diff
@@ -477,6 +466,7 @@ def __init__(
         self.losses = losses
         self.costs = costs
         self.num_classes = num_classes
+        self.has_intermediate_loss = inter_loss
```

Comment:
Does it make sense to have this be configurable for all tasks? i.e. have it as an argument to the task base class nad pass it via `super().__init__()`?